docs/chaibot-test-failure-triage.md

```
@@ -0,0 +1,433 @@
+
+### Actionable Output
+- Posts analysis in Slack threads to keep channels clean
+- Provides confidence levels for root cause identification
+- Suggests specific next steps (retest, file bug, investigate)
+- Includes direct links to logs, Sippy, and related issues
+
+## Example Output
+
+When a test failure is posted in `#opp-discussion`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)